Improvements to the run your own page. (#1360)
mastodon/documentation: content/en/user/run-your-own.md

## Why would you want to run your own Mastodon server?

There are a number of benefits to running your own Mastodon instance, including:

- Complete control over your own voice on the web, not subject to anyone else's rules or whims. Your server is your property, with your rules. Your server will exist as long as you want it to exist.
- You are *not* isolated on your own server. You can follow anyone on any other server, and they can follow you and you can exchange messages just as if you were on the same server.
- You can either limit sign-ups to be the only one on the server and run it like personal (micro)blog; maintain an invite-only community for family or friends; or, run a server anyone can sign up on - it's up to you!

{{< hint style="warning" >}}

> Please bear mind that providing a public internet service involves moderation work and community management. This kind of work becomes more complicated the larger your server grows.

Reviewer: ```suggestion
Please bear in mind that providing a public internet service involves moderation work and community management. This kind of work becomes more complicated the larger your server grows.
```

- A **domain name**. This is how you and others will access your server and how you and your users will be identified on the network.

>   **How to get**: Namecheap, Gandi, Hover, or any of the large number of domain name registrars. A domain comes with an annual cost, that varies depending on the choice of domain name.

Reviewer 2: `any of the large number of` -> `one of the many existing`

`cost,` -> `cost` or otherwise `that` -> `which`

> - A **VPS** (Virtual Private Server). Something that will run the Mastodon code, that will always be connected to the internet.

Reviewer 2: `that will` -> `which should`

>   **How to get**: DigitalOcean, Hetzner, Exoscale, Scaleway, any of the large number of hosting providers. A VPS comes with a monthly or annual cost that varies depending on hardware specifications.

Reviewer 2: `any of the large number of hosting providers` -> `or one of the many existing hosting providers`

- An **e-mail provider**. Mastodon needs to send confirmation links and various notifications through e-mail, and hosting your own SMTP server, while possible, is much more difficult to do reliably than to use a third-party provider.

>   **How to get**: Mailgun, SparkPost, Postmark, Sendgrid, any of the large number of e-mail hosting providers that expose a SMTP API. An email provider comes with a monthly or annual cost, usually based on volume of e-mails sent.

Reviewer 2: `any of the large number of e-mail hosting providers` -> `or any email hosting provider`

`expose a SMTP API` -> `expose an SMTP API`

also maybe worth trying to reconcile usage of `email` vs `e-mail` throughout the page? (no personal preference on which one we use, except perhaps a weak preference for `email` as this appears less dated and more modern)

> - (optionally) an **Object storage provider**. Mastodon can save files that you and your users upload on the hard disk drive of the VPS it runs on; however, the hard disk drive is usually not infinite, and can be difficult to upgrade later. An object storage provider gives you practically infinite metered file storage.

Reviewer 2: `An **Object Storage provider (optional).`

`hard disk drive` -> `storage drive` (many vps providers offer ssd or nvme storage)

`the hard disk drive is usually not infinite, and can be difficult to upgrade later` -> `the included storage is often limited in space, and it can be difficult to upgrade later`

`practically infinite metered file storage` -> `a way to store files separately from your VPS, which is useful if your VPS has a small storage limit`

Author: yep, agree on the email (and all of the other items suggested here) - thank you!

Managed hosting solutions are great for those who do not have experience or desire to install and maintain software. However, being in charge of all components on your own hardware gives greater control over scaling, performance and customization.

We provide a **DigitalOcean 1-Click Install Image** that you can put on a DigitalOcean droplet of your choosing, which essentially gives you everything you would otherwise have after following our installation instructions through an interactive setup wizard.

{{< caption-link url="https://marketplace.digitalocean.com/apps/mastodon" caption="Mastodon 1-Click Install Image on DigitalOcean" >}}

> The 1-Click Install does assume a single-machine setup. Mastodon scales quite well horizontally. If your needs outgrow the capacity of a single machine, Mastodon can be divided into multiple app servers, background workers, multiple Redis backends, PostgreSQL replicas -- but the 1-click Install won't cut it.

Reviewer: ```suggestion
The 1-Click Install will deploy as a single-machine setup that can be scaled by adding CPU, memory or disk capacity to your Droplet.

If the needs of your deployment outgrow those abilities, you can reconfigure to load balance across multiple app servers, use dedicated servers for background workers, or use external databases servers -- but these options are outside of the scope of the initial 1-Click Install.
```

Managed hosting solutions are great for those who do not have experience or desire to install and maintain software. However, being in charge of all components on your own hardware gives greater control over scaling, performance and customization.

> We provide a **DigitalOcean 1-Click Install Image** that you can put on a DigitalOcean droplet of your choosing, which essentially gives you everything you would otherwise have after following our installation instructions through an interactive setup wizard.

Reviewer: ```suggestion
We provide a **DigitalOcean 1-Click Install Image** that will deploy Mastodon on a DigitalOcean Droplet of your choosing, which essentially gives you everything you would otherwise have after following our installation instructions through an interactive setup wizard.
```

{{< caption-link url="https://marketplace.digitalocean.com/apps/mastodon" caption="Mastodon 1-Click Install Image on DigitalOcean" >}}

The 1-Click Install will deploy as a single-machine setup that can be scaled by adding CPU, memory or disk capacity to your Droplet.

If the needs of your deployment outgrow those abilities, you can reconfigure to load balance across multiple app servers, use dedicated servers for background workers, or use external databases servers -- but these options are outside of the scope of the initial 1-Click Install.

> If you're interested in installing, configuring and managing everything on your own, proceed here:

Reviewer 2: `configuring` -> `configuring,`
